apps/meteor/server/lib/openRoom.ts

```diff
@@ -0,0 +1,24 @@
+import { Subscriptions } from '@rocket.chat/models';
+
+import { notifyOnSubscriptionChangedByRoomIdAndUserId } from '../../app/lib/server/lib/notifyListener';
+
+export async function openRoom(userId: string, roomId: string) {
+	check(userId, String);
+	check(roomId, String);
+
+	if (!roomId) {
+		throw new Meteor.Error('error-invalid-room', 'Invalid room');
+	}
+
+	if (!userId) {
+		throw new Meteor.Error('error-invalid-user', 'Invalid user');
```

Comment:
shouldn't these be plain Errors? for the meteor method wrap Error in Meteor.Error?

Reply:
also I don't think these functions should be responsible for these validations. All callers of this should have validated then passed. Which is already happening so these checks here are unnecessary.